src/main/resources/assets/js/components/Marathon.jsx

```diff
@@ -171,6 +227,10 @@ define([
     },
 
     handleModalDestroy: function() {
+      Backbone.history.navigate("#"+this.state.activeTabId, { trigger:true });
+    },
+
+    modalDestroy: function() {
```

Comment:
Makes sense. But not triggering the new route I could reduce it to one function.
this.props.router.navigate("#"+this.state.activeTabId, { trigger: false });